Makefile

```diff
@@ -87,10 +87,7 @@ clean:
 
 generate-examples: latest-ksonnet $(EXAMPLE_OUTPUT)
 
-examples/quickstart/components/%.yaml: examples/ksonnet/components/%.jsonnet
-	$(KUBECFG_CMD)
-
-examples/quickstart/%.yaml: examples/ksonnet/%.jsonnet
+examples/quickstart/%.yaml: examples/ksonnet/%.jsonnet examples/ksonnet/components/*.jsonnet
```

Comment:
not sure if you can call the target `$(EXAMPLE_OUTPUT)` but now that there's only one YAML file, it is clearer if use `examples/quickstart/aggregate.yaml` instead of `examples/quickstart/%.yaml`

Reply:
Doesn't hurt atm, and I'm going to revamp it some more with a release target.

Reply:
ok sounds good